a10_octavia/controller/worker/tasks/handler_server.py

```diff
@@ -140,3 +140,54 @@ def execute(self, member, vthunder, pool):
         except Exception as e:
             LOG.error(str(e))
             LOG.info("Error occurred")
+
```

Comment:
Note : This got added while resolving conflicts

Reply:
The added space should be there. Always should have 2 spaces between classes.

Reply:
The comment is not about spaces. Just made a note for ourselves that the entire set of classes has got added while you were resolving conflicts for this file.

Reply:
Ohhh you meant `MemberUpdate`. Yeah let me remove that.